iroha/src/sumeragi.rs

```diff
@@ -103,6 +107,11 @@ impl Sumeragi {
 
     /// the leader of each round just uses the transactions they have at hand to create a block
     pub async fn round(&mut self, transactions: Vec<AcceptedTransaction>) -> Result<(), String> {
+        self.events_sender
+            .send(crate::event::Occurrence::Updated(
+                crate::event::Entity::Time,
+            ))
+            .await;
```

Comment:
Maybe we can spawn a separate task in `lib` to update time? Or you want to track time when the sumeragi tries to create a block? If it's the later, I guess you should add a comment here or provide more descriptive naming.